pkg/apis/externaldns/types.go

```diff
@@ -29,57 +29,59 @@ var (
 
 // Config is a project-wide configuration
 type Config struct {
-	Master             string
-	KubeConfig         string
-	Sources            []string
-	Namespace          string
-	FQDNTemplate       string
-	Compatibility      string
-	PublishInternal    bool
-	Provider           string
-	GoogleProject      string
-	DomainFilter       []string
-	AWSZoneType        string
-	AzureConfigFile    string
-	AzureResourceGroup string
-	CloudflareProxied  bool
-	Policy             string
-	Registry           string
-	TXTOwnerID         string
-	TXTPrefix          string
-	Interval           time.Duration
-	Once               bool
-	DryRun             bool
-	LogFormat          string
-	MetricsAddress     string
-	LogLevel           string
+	Master                 string
+	KubeConfig             string
+	Sources                []string
+	Namespace              string
+	FQDNTemplate           string
+	Compatibility          string
+	PublishInternal        bool
+	Provider               string
+	GoogleProject          string
+	DomainFilter           []string
+	AWSZoneType            string
+	AzureConfigFile        string
+	AzureResourceGroup     string
+	CloudflareProxied      bool
+	Policy                 string
+	Registry               string
+	TXTOwnerID             string
+	TXTPrefix              string
+	Interval               time.Duration
+	Once                   bool
+	DryRun                 bool
+	LogFormat              string
+	MetricsAddress         string
+	LogLevel               string
+	SourceAnnotationFilter string
```

Comment:
I tried to maintain a cosistent ordering. Let's add this under/behind the source's `namespace` parameter. So, in all the structs, tests and kingpin definitionions.